book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add.rentals(rentals)
+    @rentals.push(rentals)
+    rentals.book = self
+  end
```

Comment:
- [ ] In order to add a rental, you need a person and a date. It can be something similar to this
```
  def add_rental(person, date) 
    Rental.new(date, self, person)
```